balancer/rls/balancer.go

```diff
@@ -77,6 +78,28 @@ var (
 	clientConnUpdateHook = func() {}
 	dataCachePurgeHook   = func() {}
 	resetBackoffHook     = func() {}
+
+	defaultTargetPicksMetric = estats.RegisterInt64Count(estats.MetricDescriptor{
+		Name:        "grpc.lb.rls.default_target_picks",
+		Description: "EXPERIMENTAL. Number of LB picks sent to the default target.",
+		Unit:        "pick",
+		Labels:      []string{"grpc.target", "grpc.lb.rls.server_target", "grpc.lb.rls.data_plane_target", "grpc.lb.pick_result"},
+		Default:     false,
```

Comment:
Nit: Ignore the `Default` field since `false` is the zero value for `bool`?

Reply:
I'd prefer to keep this explicit since a user now knows they need to configure it on through the OTel API. All of these non per call metrics are experimental and default off for now.

Reply:
Ack.